### ISSUE TITLE
Ratelimit is not defined

### ISSUE THREAD

**Issue by DBMfan** [Sank6/Discord-Bot-List#561]
Hello, I'm getting this error:
```
[2021-07-12 21:46:36] Error [ERR_UNHANDLED_ERROR]: Unhandled error. ('Uncaught Promise Error: \n' +
[2021-07-12 21:46:36]   'ReferenceError: ratelimit is not defined\n' +
[2021-07-12 21:46:36]   '    at /app/src/routes/api/auth/stats.js:18:127\n' +
```

Is there anyway to fix it?

**Comment by reisxd** [comment]
> Hello, I'm getting this error:
> 
> ```
> [2021-07-12 21:46:36] Error [ERR_UNHANDLED_ERROR]: Unhandled error. ('Uncaught Promise Error: \n' +
> [2021-07-12 21:46:36]   'ReferenceError: ratelimit is not defined\n' +
> [2021-07-12 21:46:36]   '    at /app/src/routes/api/auth/stats.js:18:127\n' +
> ```
> 
> Is there anyway to fix it?

Did you modify your `stats.js` file?

**Comment by DBMfan** [comment]
No

**Comment by reisxd** [comment]
> No

Because i dont see anything about ratelimit on that file...Weird

**Comment by Sank6** [comment]
Fixed 353e1fd6dcdea5a0b6e84ed477de70dacc1cf409
